### ISSUE TITLE
[Question] My T-Friendly marks are never pushed over to the main extension.

### ISSUE THREAD

**Issue by Meep560** [shinigami-eyes/shinigami-eyes#137]
As the title says whenever I mark somebody as trans friendly it never seems to go to other devices. I know it's cached locally for me but am I doing something wrong? Is my definition of trans friendly wrong or is the human reviewer not great at their job. I honestly want to know because I want to support the extension and I don't want to waste people's time if I'm doing more harm then good. As an example let me take what the YouTube channel Mister Technodad (https://www.youtube.com/@MrTechnodad) said on International Trans Day of Visibility this year in his community tab.

Part 1 reads
"Hey everyone! Mr. Technodad here. I just want to wish everyone everywhere, and especially trans people, a happy Trans Day of Visibility.

First and foremost, trans acceptance is important because acceptance of different people is a requirement for a just and civil society. Treating everyone with acceptance and tolerance is a basic requirement for everyone.* If you're judging people by attributes that are part of who they are, you are simply increasing the unhappiness in the world. Please don't do that.

But enough of that larger context stuff; let us simply celebrate trans people today for having the courage to be who they are in a world that is sometimes less accepting than it ought to be.

How Many Genders Are There?

I was having a conversation with some young people in a bar in Boston a week ago. How many genders are there? The fact is, everyone is different and everyone is unique. The things that make up gender are varied and diverse, and they don't imprint the same way on any two people. We are all unique. The answer to how many genders there are is eight billion. There are eight billion genders! My gender is "Mr. Technodad" and my pronouns are "he/him".

Thus, today I pay my respects to all who use "they/them" (or whatever else as appropriate)!

Salute!

-- Mr. Technodad

* Except the intolerant. See: https://en.wikipedia.org/wiki/Paradox_of_tolerance "

Part 2 reads
"Trans Day of Visibility, Part 2

Hey everyone. Mr. Technodad with a community post in followup to what is currently my most-commented community post, commemorating Transgender Day of Visibility.

Most of the comments have been supportive, whether coming from trans people or from other members of the lgbtq community or from other people who are cis het like myself.

Some minority of the comments have been good examples of the kind of hate-filled rhetoric that are why we need to have a Trans Day of Visibility in the first place.

I want to address the complaints that Easter happens to fall on Trans Day of Visibility this year.

There is no biblical record or scholarly agreement on the date of Christ's resurrection. There isn't agreement on the year, either: there are arguments for both 30 AD and 33 AD. There isn't even agreement on what calendar to use! The Greek Orthodox and Eastern Orthodox churches calculate Easter according to the Julian calendar. According to the Julian calendar, Easter this year will be on May 5th. As a Greek, I will be happy to greet you with "Χριστός ἀνέστη" on that day. In the meantime, as a decent human being and understanding that many people celebrate Easter by the Gregorian Calendar, I will also wish you a belated "Happy Easter".

Trans Day of Visibility falls on March 31st every year, and has since its creation in 2009. This year, most people calculate Easter as also falling on March 31st.

Last year, Trans Day of Visibility fell on March 31st, because it does that every year. Last year most people calculated Easter as falling on April 9.

To the people who are saying "Biden planned this whole thing", I will point out that President Biden issued a statement affirming TDoV as March 31 in 2021, 2022, 2023, and 2024. He has also issued an Easter statement on Easter day every year as well. In his 2023 Easter statement, he wrote, "Today, we hold close the Easter message that nothing – not even death – can match the power of faith, hope, and love. And we remember Jesus’ sacrifice and recommit ourselves to love God with all our hearts and to love our neighbors as ourselves, embracing His call to treat one another with compassion."

When he says, "to love God with all our hearts and to love our neighbors as ourselves" he's quoting someone. Do you know who?

I was interested to learn that President Biden also makes a statement on Orthodox Easter every year.

Last year, President Biden issued a proclamation that April 9, 2023 shall be designated "National Former Prisoner Of War Recognition Day." That's right: Easter! Easter 2023 was proclaimed by the president as National Former Prisoner Of War Recognition Day, just like every April 9th has been so declared since 1988. Apparently, Ronald Reagan planned this whole thing "to try to displace God's day on Earth" as a few of you would have it.

If you didn't protest Reagan/Biden involvement in National Former Prisoner Of War Recognition Day last year every bit as strongly as you're protesting TDoV this year, then your actions give you away. You aren't motivated by respect for Easter, you're motivated by transphobia. You're not loving thy neighbor as thyself.

To those who are threatening to unsubscribe for my support of lgbtq+ people, please do. I am cultivating a community based on love and compassion, and if you are committed to attacking people because they are different from you then you're not welcome here. I'd rather speak to a community of ten open minded compassionate young people than I would ten million transphobes.

And if you continue to post intolerance on my community posts, know that I have crafted a Ban Hammer from netherite and it is unbreakable. I will tolerate anything except intolerance.

To those who are taking a flat-earth-science approach to politics, it seems like a skill issue. Get good, noobs!

To everyone else: VOTE! Especially the younger folks."

All of this seems like enough to mark somebody as Trans friendly. Am I wrong and should I have not marked it this way? Because it hasn't been pushed over to the main extension. I'm not going to provide long examples because I don't want to waste your time but other people I have marked as trans friendly are:
The Click: https://www.youtube.com/@TheClick who frequently  makes videos on LGBTQ/trans topics dedicating whole videos. He has even collaborated with Jammidodger ( https://www.youtube.com/@Jammidodger ) who I don't watch but is marked as trans friendly.
And 
Azeal: https://www.youtube.com/@Azeal Who I can understand not being marked as trans friendly because his whole thing is giving everybody a voice except he has explicitly stated that he wouldn't give any bigoted people a voice on his channel. He has stated being trans supportive time and time again. He's also interviewed a lot of trans people including a trans male athlete to try dispel the falsity that trans people shouldn't be allowed in sports. I might as well link every video in which interviews a trans person here so those videos would be from oldest to newest:
https://www.youtube.com/watch?v=RrpbOeGz1VU
https://www.youtube.com/watch?v=ogLIcpKPAYI
https://www.youtube.com/watch?v=PQZtqKDNV1E
https://www.youtube.com/watch?v=xrsoselqh7w
https://www.youtube.com/watch?v=JylCl4viih4
https://www.youtube.com/watch?v=CvrHTbdo9ps
https://www.youtube.com/watch?v=hB_GhltWNAI
https://www.youtube.com/watch?v=EHXEaKGjDfk
https://www.youtube.com/watch?v=_iipAOtNlds
In all of these he has constantly stood for trans rights and tried to give a voice to trans people with all kinds of diffrent experiences.

In summary am I wrong for marking these channels as trans friendly? I really don't believe I am but I don't want to give anybody extra work if I don't understand the guidelines that are set. Or is it that the human reviewer and/or the algorithm just decided that these very trans supportive channels should remain neutral.
